packages/ubuntu_desktop_installer/lib/main.dart

```diff
@@ -9,7 +10,33 @@ import 'app.dart';
 import 'keyboard_model.dart';
 import 'l10n/app_localizations.dart';
 
+class AppTheme extends ValueNotifier<ThemeMode> {
```

Comment:
Would it be possible to place `AppTheme` in a different file [*] to avoid importing 'main.dart' in the app code? It just feels somehow off to import `main.dart` in the app code, and technically, we could later even have multiple main variants for different configurations. :wink:

[*] Perhaps even a whole new file, something dedicated for theming, where we can stick app-wide padding and spacing constants etc.

Reply:
Thanks @jpnurmi :) Done, you are right! Ofc you are ;)